pkg/registry/registry.go

```diff
@@ -93,12 +123,19 @@ func (r *registry) LastLayer(ctx context.Context, entry string) (v1.Layer, error
 	if hash := strings.Split(entry, "@"); len(hash) > 1 {
 		repo = hash[0]
 	} else if tag := strings.Split(entry, ":"); len(tag) > 1 {
+
 		repo = tag[0]
 	}
 
-	manifest, err := crane.Manifest(entry)
+	var registryAuths []crane.Option
+	if auth.Email != "" && auth.Auth != "" {
```

Comment:
Isnt the mail optional? Only `auth` is required AFAIK

Reply:
Changed!